src/ui.js

```diff
@@ -8,6 +8,8 @@ import {
 } from './config.js';
 import './ui.css';
 
+const ytaf_version = __YTAF_VERSION__;
```

Comment:
This variable is only used once. You should just directly use the global in the template string.